package.json

```diff
@@ -22,20 +22,14 @@
     "update-aws-cdk": "ncu \"/^@?aws-cdk(/.*)?$/\" --upgrade --deep && npm install"
   },
   "devDependencies": {
-    "@guardian/eslint-config-typescript": "^0.5.0",
+    "@guardian/eslint-config-typescript": "^0.6.0",
     "@types/aws-lambda": "^8.10.76",
     "@types/git-url-parse": "^9.0.0",
     "@types/jest": "^26.0.23",
     "@types/node": "15.6.1",
-    "@typescript-eslint/eslint-plugin": "^4.26.0",
-    "@typescript-eslint/parser": "^4.26.0",
     "cz-conventional-changelog": "^3.3.0",
     "eslint": "^7.27.0",
-    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-custom-rules": "file:eslint",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.23.2",
-    "eslint-plugin-prettier": "^3.4.0",
```

Comment:
These are all direct dependencies of `@guardian/eslint-config-typescript` now and become transitive in this repository 🎉 .